docs/site/angular-auth-oidc-client/docs/documentation/configuration.md

```diff
@@ -12,8 +12,8 @@ Prior to using the library, you must configure it with the appropriate values fo
 You can pass the static config with the `config` property into the `forRoot()` method like this
 
 ```ts
-import { NgModule } from '@angular/core';
-import { AuthModule } from 'angular-auth-oidc-client';
+import { NgModule } from '@Angular/core';

```

Comment:
Imports are lowercase, as the folder in node_modules is

Reply:
Whoops, good catch. Had a bad replace-all there.